packages/internal/bridge/sdk/src/lib/utils.test.ts

```diff
@@ -24,4 +33,233 @@ describe('utils', () => {
       expect(result).toEqual('devnet');
     });
   });
+
+  describe('isWithdrawNotWrappedIMX', () => {
+    it('should return false for non-wrapped IMX but not withdraw', () => {
```

Comment:
nit: can we use a diff term for "non-wrapped"/"not-wrapped" 😅 ... this is def subjective, but i find it confusing ... maybe "native"?

Reply:
Yeah lol good point